books/refgram/src/logic/explicit_binding.md

```diff
@@ -92,14 +92,14 @@ The above example can thus be shortened as _mi dona [va be mian buri __ba__]_.
 
 ## Right place and chaining selection
 
-When it is only needed to bind one or 2 places of a predicate, using __VI/FI__
-and arguments lists is quickly verbose. For that reason, predicates can be
-prefixed with particles of familly __SI__ (all particles starting with _s-_),
-which overrides the chaining behavior.
+When it is only needed to bind one or two places of a predicate, using __VI/FI__
+and argument lists quickly becomes verbose. For that reason, predicates can be
```

Comment:
Those are lists of (possibly multiple) arguments. There should be no plural at "arguments"?